bigbluebutton-tests/playwright/.env.template

```
@@ -1,6 +1,7 @@
 # meeting credentials
 BBB_URL=""                        # https://DOMAIN_NAME/bigbluebutton/api
 BBB_SECRET=""
+CHECKSUM=""                       # Force a specific checksum algorithm. Useful for when the secret length is not a sha algorithm length
```

We could add the possible inputs:
```suggestion
CHECKSUM=""                       # (sha1 | sha256 | sha512) Force a specific checksum algorithm. Useful for when the secret length is not a sha algorithm length
```

---

Good suggestion.